components/compose/cfr/README.md

```diff
@@ -0,0 +1,49 @@
+# [Android Components](../../../README.md) > Compose > Tabs tray
+
+A standard Contextual Feature Recommendation popup using Jetpack Compose.
+
+## Usage
+
+```kotlin
+CFRPopup(
+    anchor = <View>,
+    properties = CFRPopupProperties(
+        popupWidth = 256.dp,
+        popupAlignment = INDICATOR_CENTERED_IN_ANCHOR,
+        popupBodyColors = listOf(
+            ContextCompat.getColor(context, R.color.color1),
+            ContextCompat.getColor(context, R.color.color2)
+        ),
+        dismissButtonColor = ContextCompat.getColor(context, R.color.color3),
+    ),
+    onDismiss = { <method call> },
+    text = {
+        Text(
+            text = stringResource(R.string.string1),
+            style = MaterialTheme.typography.body2,
+        )
+    },
+    action = {
```

Comment:
Self-note: Comparing to what is in Focus, it looks like `action` is the new API addition with the added `properties` above too.

```kotlin
class CFRPopup(
    private val container: View,
    private val text: String,
    private val anchor: View,
    private val properties: CFRPopupProperties = CFRPopupProperties(),
    private val onDismiss: () -> Unit = {},
)
```